### PR TITLE
Allow specifying config_file and node_name

### PR DESCRIPTION
**Pull request by timrobotson**
- Add parameters to allow the user of this node to specify their own
config file and node name.
- Add env var to make logging output stream to terminal instead of buffering. 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: lex_node/launch/lex.launch.py [hunk 1]
```diff
@@ -18,15 +18,30 @@
...
+            default_value=config_file_path
+        ),
+        Node(
+            package='lex_node',
+            node_executable='lex_node',
+            node_name=LaunchConfiguration('node_name'),
+            parameters=[LaunchConfiguration('config_file')],
+            output='screen',
+            additional_env={
+                'RCUTILS_CONSOLE_STDOUT_LINE_BUFFERED': '1'
```
Comment (ryanewel) [inline lex_node/launch/lex.launch.py]: Why is this something we're adding to this Cloud extension and none of the others?

Comment (timrobotson) [reply lex_node/launch/lex.launch.py]: Without it the Lex output was being buffered until the app was killed. Do the other cloud extensions have some other workaround for this?

Comment (ryanewel) [reply lex_node/launch/lex.launch.py]: That makes sense, I'm not sure how it's working for the other CEs

Comment (timrobotson) [reply lex_node/launch/lex.launch.py]: Have changed it to https://github.com/ros2/demos/blob/388a07d96286f6a027e7403a5eac2f618257dae3/logging_demo/src/logging_demo_main.cpp#L24 which will be more reliable (so will work using other launch files etc).

**Pull request merged**